device/dell/x86_64-dell_z9100_c2538-r0/media_settings.json

```diff
@@ -0,0 +1,751 @@
+{
+    "GLOBAL_MEDIA_SETTINGS": {
+        "1-32": {
+            "AMPHENOL-NDARHF-M201": {
+                "preemphasis": {
+                    "lane0":"0x006808",
+                    "lane1":"0x006808"
```

Comment:
why only two lanes here, but others have four lanes?

Reply:
@lguohan These are 2x50G DAC media and thus will have only two lanes per port